docs/fallbacks.rst

```diff
@@ -16,16 +16,21 @@ or if none of the dialogue policies predict an action with confidence higher tha
    from rasa_core.policies.keras_policy import KerasPolicy
    from rasa_core.agent import Agent
 
-   fallback = FallbackPolicy(fallback_action_name="utter_default",
+   fallback = FallbackPolicy(fallback_action_name="action_fallback",
                              core_threshold=0.3,
                              nlu_threshold=0.3)
 
    agent = Agent("domain.yml",
                   policies=[KerasPolicy(), fallback])
 
 
-``action_listen`` will always be predicted after the fallback action, meaning
-the bot will wait for the next user input.
+``action_fallback`` is a default action in Rasa Core, which will send the
+``utter_default`` template message to the user. Make sure to specify this template
+in your domain file. It will also revert back to the state of the conversation
+before this action was executed, so that it will not influence the prediction of
+future actions. You can take a look at the source of the action below:
```

Comment:
Let's add a note that you can also create your own action and use it as the fallback action.

Reply:
and make sure the user knows, that they can not name their fallback action `action_fallback`

Reply:
probably better to name the default fallback `action_default_fallback` to avoid collisions with user actions